GUIs/ScriptRunnerPyQt5_GUI/scripFunc/scripWINONLY.py

```diff
@@ -550,63 +757,102 @@ def lsmemofgru(self):
     def lsgrusanusrin(self):
         pass
 
-    # @staticmethod
     def chngpasswdofall(self):
-
-        # TODO: This command as of July 7th, 2020 is broken. More on the issue can be found here: https://github.com/vipersniper0501/CP_Scripts2/issues/48
-
         class changepasswordofall(QDialog, Ui_chngpass):
-            def __init__(self, parent=None):
+            def __init__(self, parent = None):
                 super(changepasswordofall, self).__init__(parent)
                 self.setWindowIcon(QtGui.QIcon(':/Pictures/images/cup2.png'))
                 self.setupUi(self)
                 self.EXECUTE()
 
             def EXECUTE(self):
                 def findnames():
-                    command = 'Get-LocalUser'
-                    EXEC = sub.Popen(["powershell", "& {" + command + "}"], stdout=sub.PIPE)
-                    stdout, _ = EXEC.communicate()
-                    output = stdout.decode("utf-8")
-                    output = output.split('\n')
-                    print(output)
-                    n = []
-                    for i in range(0, len(output)):
-                        if 'True' in output[i]:
-                            # print('found in ' + str(i))
+                    names = []
+                    try:
+                        print('Detecting Users On System...')
+                        # Local users are added to a list of names
+                        EXEC = sub.Popen(["powershell", "& {net localgroup users}"],
```

Comment:
Codacy found an issue: [subprocess call - check for execution of untrusted input.](https://app.codacy.com/manual/vipersniper0501/CP_Scripts2/pullRequest?prid=5927419)